app/models/spotlight/custom_field.rb

```diff
@@ -50,12 +50,12 @@ def configured_to_display?
         end
     end
 
-    def solr_field
-      if field && field.starts_with?(solr_field_prefix)
+    def solr_field(document_model)
```

Comment:
:ok_hand: I think we've run into that before too. I'll check if there's anything else we can do.